src/NPacMan.Game/Actions.cs

```diff
@@ -0,0 +1,139 @@
+﻿using Automatonymous;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace NPacMan.Game
+{
+    internal static class Actions
```

Comment:
This class should be the only place where game state can be changed???

Reply:
Yeah, That'd be nice 👍